cmds/core/netcat/netcat.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 const usage = "netcat [go-style network address]"
+const bufSize = 1500
```

Comment:
it's probably worth finding what package defines ETHERMTU and using that?

Reply:
I was able to find this constant, but build constraints for it is very severe. It's defined for netbsd or [openbsd](https://go.dev/src/cmd/vendor/golang.org/x/sys/unix/zerrors_openbsd_386.go) but not for Darwin.

In openbsd they don't really care and use 2048 https://github.com/openbsd/src/blob/ec5fc28303615d86d52eabb43d10128e2b7aadb5/usr.bin/nc/netcat.c#L599 if understand it correctly. 

So maybe we can make it 2048 like openbsd?

Reply:
2048? I guess. Wow. I think just leave a comment here about what it is, and that it is not universal. MTU? On most OSX interfaces MTU is 1500. And it could be a bit more but 1500 is the convention. A mess.